pkg/traceability/healthcheck_test.go

```diff
@@ -0,0 +1,87 @@
+package traceability
```

Comment:
I believe this should not be commented out

Reply:
you right, will fix